grype/db/internal/gormadapter/open.go

```diff
@@ -29,7 +36,10 @@ func Open(path string, write bool) (*gorm.DB, error) {
 	}
 
 	if !write {
-		connStr += "&immutable=1"
+		// &immutable=1&cache=shared&mode=ro&_journal_mode=WAL
+		for _, o := range readOptions {
```

Comment:
nit: could we remove the `&` from the list elements up above and add them in explicitly here?